client/utils/use-confirm-navigation.js

```diff
@@ -28,9 +28,12 @@ const useConfirmNavigation = ( displayPrompt ) => {
 			event.preventDefault();
 			event.returnValue = '';
 		};
+
+		// eslint-disable-next-line @wordpress/no-global-event-listener
```

Comment:
I'm disabling the rule here in this file as there's no element that we can use to get the `defaultView` object.